src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlExceptionTest.cs

```diff
@@ -4,6 +4,7 @@
 
 using System;
 using System.IO;
+using System.Runtime.Serialization;
```

Comment:
unnecessary

Reply:
thanks I thought it was removed in previous push. I will remove it right away.